tests/test_worker_palavras_raw.py

```diff
@@ -37,6 +39,7 @@ def test_should_run_only_if_language_is_portuguese(self):
             refreshed_document = self.collection.find_one({'_id': doc_id})
             self.assertEqual(refreshed_document['palavras_raw_ran'], False)
 
+    @skipIf(not palavras_raw.palavras_installed(), 'palavras software is not installed')
     def test_palavras_not_installed(self):
```

Comment:
I think this test should run even if palavras is not installed. Specially so, even, because with it we guarantee that palavras not being present is not a problem.